### ISSUE TITLE
FFXIV non-standard Steam locations

### ISSUE THREAD

**Issue by JacoG-RH** [HereInPlainSight/gshade_installer#3]
I have a very crude script that looks through all Steam install locations to make some symlinks. It's ugly, but maybe you can use some of what's there to make the XIVInstall find FFXIV no matter where it lives?

https://github.com/screwylightbulb/SteamCompatLink

I guess I could submit a PR also, but I don't want to mess with your script. My bash is not great.

EDIT: I have made the change and tested on my own and my partner's systems. Below is the chunk of code that changed:

```
  # create array of steamdirs first
  steamdirs=("$HOME/.steam/steam")
  readarray -t steamdirs < <(cat ~/.steam/steam/steamapps/libraryfolders.vdf | grep "/" | awk '{ print $2  }' | sed s/\"//g)

  # find the one containing ffxiv
  for dir in "${steamdirs[@]}"; do
    compatdir="${dir}/steamapps/compatdata/39210/pfx"
    if [ -d "${compatdir}" ]; then
      ffxivDir="${dir}"
    fi
  done

  if [ -d "${ffxivDir}/steamapps/common/FINAL FANTASY XIV Online/game/" ] && [ -d "${ffxivDir}/steamapps/compatdata/39210/pfx" ]; then
    WINEPREFIX="${ffxivDir}/steamapps/compatdata/39210/pfx"
    gameLoc="${ffxivDir}/steamapps/common/FINAL FANTASY XIV Online/game/"
    printf "\nSteam install found!\n\tPrefix location: %s\n\tGame location: %s\n" "$WINEPREFIX" "$gameLoc"
    if (yesNo "Install? "); then
        if ( ! yesNo "Use $gapi instead of dxgi?  If you are having issues with GShade when using other overlays (Steam or Discord, for instance), you may wish to try dxgi mode instead." ); then gapi=dxgi; fi
      printf "\nInstalling...  "
      installGame
      printf "Complete!\n"
    fi
  fi
```

**Comment by HereInPlainSight** [comment]
This seems good!  The only issue I had in testing is that `readarray` seems to replace the array rather than append it, so it was ignoring the default `~/.steam/steam/` location.  Other than that the only changes I made were to camel case and make a variable slightly more unique, and compacted your if statement a bit.  I'm a little OCD.

Added a Contributor's section to the README.  :)

(I'm sure there's a correct way to close referencing the commit directly, but I am not git-savvy.)
